docs/ecosystem/changelog.md

```diff
@@ -3,27 +3,32 @@ id: changelog
 title: Changelog and roadmap
 ---
 
-## Changelog
+## Ory Changelog
+
+:::info
+
+Read the changelog for the Ory ecosystem at [changelog.ory.sh](https://changelog.ory.sh/) !
+
+:::
 
 > Running security-critical open source technology in a self-service format requires vigilance. We make your life easier by
 > announcing important software updates via the [Ory Security Newsletter](http://eepurl.com/di390P). Never miss an update and
 > **[sign up now to important release updates!](http://eepurl.com/di390P)**
 
-:::warning
+:::tip
 
-Changelogs contain important information about the changes and breaking changes introduced in the code. Please read the relevant
-project's changelog before you [upgrade](./upgrading.md) to a new version.
+Looking for support? Please contact us at support@ory.sh!  
+Ory offers support agreements for self-hosted Ory software.
```

Comment:
Make this specifically about the getting help with upgrades of selfhosted?

Reply:
Not sure what you mean, it is specifically for support for self-hosted, no?

Reply:
Looks like I can't read